docs/cloud-workload-security/agent_expressions.md

```diff
@@ -36,7 +36,9 @@ Triggers are events that correspond to types of activity seen by the system. The
 | `capset` | Process | A process changed its capacity set | 7.27 |
 | `chmod` | File | A file’s permissions were changed | 7.27 |
 | `chown` | File | A file’s owner was changed | 7.27 |
+| `delete_module` | Kernel | A kernel module was deleted | 7.35 |
```

Comment:
`unload_module` | Kernel | A kernel module was unloaded

Reply:
I think the doc just needs to be regenerated